expath-pkg.xml

```diff
@@ -1,4 +1,5 @@
+
 <package xmlns="http://expath.org/ns/pkg" name="http://srophe.org/usaybia" abbrev="usaybia" version="0.7.1-dev" spec="1.0">
```

Comment:
@wsalesky FYI I think the package version here is the only value that _has_ to remain different between master and development for our workflows to funciton. So we need to not merge this file back into master or else just change it again once we do.